src/features/CourtCaseDetails/Sidebar/TriggersAndExceptions.tsx

```diff
@@ -59,46 +59,48 @@ const TriggersAndExceptions = ({ courtCase, aho, user, onNavigate, canResolveAnd
 
   return (
     <div className={`${classes.sideBar} triggers-and-exceptions-sidebar`}>
-      <ConditionalRender isRendered={user.hasAccessTo[Feature.CaseDetailsSidebar]}>
+      <ConditionalRender isRendered={user.hasAccessTo[Permission.CaseDetailsSidebar]}>
         <Tabs>
           <TabList>
-            <ConditionalRender isRendered={user.hasAccessTo[Feature.Triggers]}>
+            <ConditionalRender isRendered={user.hasAccessTo[Permission.Triggers]}>
               <Tabs.Tab
                 id="triggers-tab"
                 className={classes.tab}
-                onClick={() => setSelectedTab(Feature.Triggers)}
-                selected={selectedTab === Feature.Triggers}
+                onClick={() => setSelectedTab(Permission.Triggers)}
+                selected={selectedTab === Permission.Triggers}
               >
                 {`Triggers`}
               </Tabs.Tab>
             </ConditionalRender>
 
-            <ConditionalRender isRendered={user.hasAccessTo[Feature.Exceptions]}>
+            <ConditionalRender
+              isRendered={user.hasAccessTo[Permission.Exceptions] && user.hasAccessTo[Permission.ExceptionsEnabled]}
```

Comment:
since we're doing something not very obvious unless you know the context here, I think it's worth adding a comment explaining why we have these two separate permissions

Reply:
I agree, maybe it shouldn't be part of permissions but we could just use `!!user.featureFlags?.exceptionsEnabled` since permissions are part of the business logic and feature flags are just temporary.